rideshare/Controllers/NewRideViewController.swift

```diff
@@ -59,34 +59,53 @@ class NewRideViewController: UIViewController, UINavigationControllerDelegate, S
     }
     
     @IBAction func onPostRideButton(_ sender: Any) {
-        print("\n\nposting ride")
-        let ride = PFObject(className:"Rides")
-        ride["departureLocation"] = departureLocationField.text
-        print(departureLocationCL as Any)
-        ride["departureLocationLat"] = departureLocationCL!.placemark.coordinate.latitude
-        ride["departureLocationLong"] = departureLocationCL!.placemark.coordinate.longitude
-        ride["arrivalLocation"] = arrivalLocationField.text
-        ride["arrivalLocationLat"] = arrivalLocationCL!.placemark.coordinate.latitude
-        ride["arrivalLocationLong"] = arrivalLocationCL!.placemark.coordinate.longitude
         
-        let formatter = DateFormatter()
-        formatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ss"
-
-        let departureDateTime = formatter.string(from: departureDatePicker.date)
-        let arrivalDateTime = formatter.string(from: arrivalDatePicker.date)
-        ride["departureDateTime"] = departureDateTime
-        ride["arrivalDateTime"] = arrivalDateTime
-        
-        ride["driverId"] = PFUser.current()
-        ride["rideDetails"] = rideDetailsTextView.text
-        ride.add(PFUser.current() as Any, forKey: "riders")
+        if (departureLocationField.text!.isEmpty || arrivalLocationField.text!.isEmpty || rideDetailsTextView.text!.isEmpty || arrivalLocationCL == nil || departureLocationCL == nil) {
```

Comment:
I could see either way making sense, but do you think we should require that the user fills out `rideDetailsTextView`? Also, I think we should remove the arrival time, which could be a separate issue. Having the MapKit show the amount of time for the trip makes it kind of redundant information that might conflict if the poster doesn't actually know how long it will take. lmk what you think and we can open a new issue to refactor that if you think that makes sense.